protobuf/v1/replica.proto

```diff
@@ -130,3 +132,18 @@ message ListReplicaSnapshotsResponse {
 message DeleteReplicaSnapshotRequest {
   string snapshot_uuid = 1; // snapshot to be deleted
 }
+
+/// Request Parameters for Creating Snapshot Clone.
+message CreateSnapshotCloneRequest {
+  string snapshot_uuid = 1; // uuid of the snapshot
+  string clone_name = 2; // snapshot clone name
```

Comment:
Shouldn't this also be a uuid?

Reply:
Current SPDK Clone API doesn't support custom UUID. To start with, We wanted to use the existing SPDK API which accept 
1. Source snapshot lvol from which Clone can be created
2. New Clone Volume Name.
Clone lvol UUID will be generated by SPDK.

Reply:
ok. Will extend the SPDK API. Will update the interface.